src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import core.basesyntax.filedata.DataConvertor;
+import core.basesyntax.filedata.DataConvertorImpl;
+import core.basesyntax.filedata.FileReader;
+import core.basesyntax.filedata.FileReaderImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.report.FileWriter;
+import core.basesyntax.report.FileWriterImpl;
+import core.basesyntax.report.ReportRecorder;
+import core.basesyntax.report.ReportRecorderImpl;
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.service.FruitShopServiceImpl;
+import core.basesyntax.storage.FruitStorage;
+import java.io.File;
+import java.util.List;
+
+public class Main {
+    public static final String fromFileName = "src/main/resources/text.csv";
```

Comment:
rename input file and also push your output file 
```suggestion
    public static final String fromFileName = "src/main/resources/input.csv";
```